lib/Dialect/SV/SVOps.cpp

```diff
@@ -738,13 +752,12 @@ static void printCaseZOp(OpAsmPrinter &p, CaseZOp op) {
   }
 }
 
-static LogicalResult verifyCaseZOp(CaseZOp op) {
-  // Ensure that the number of regions and number of case values match.
-  if (op.casePatterns().size() != op.getNumRegions())
-    return op.emitOpError("case pattern / region count mismatch");
-  return success();
+ParseResult CaseZOp::parse(OpAsmParser &parser, OperationState &result) {
+  return CaseLikeOp::parse(parser, result);
 }
 
+void CaseZOp::print(OpAsmPrinter &p) { CaseLikeOp::print1(p); };
```

Comment:
The more canonical way of handling this is to have the common code in a static function in the file, rather than a member function.